nsxt/resource_nsxt_lb_pool.go

```diff
@@ -189,9 +189,15 @@ func getPoolMemberGroupSchema() *schema.Schema {
 					ValidateFunc: validation.StringInSlice(ipRevisionFilterTypeValues, false),
 					Default:      "IPV4",
 				},
+				"limit_ip_list_size": &schema.Schema{
+					Type:        schema.TypeBool,
+					Description: "Specifies whether to limit pool members. If false, dynamic pool can grow up to the load balancer max pool member capacity.",
+					Optional:    true,
+					Default:     false,
```

Comment:
should  the default be false? This means that if the user forgets to set it (or runs older config), the size will be ingnored.

Reply:
Thinking about it again - I guess it should be false.